CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# dbt_twilio v0.4.0
+
+## Features
+- Added the ability to disable models related to the `USAGE_RECORD` source table. Refer to the [README](https://github.com/fivetran/dbt_twilio?tab=readme-ov-file#step-4-enablingdisabling-models) for more details.
```

Comment:
Can we include the variable here in the CHANGELOG so customers are immediately aware of it. We can/should still link to the README, but would be good to share the variable as well.

Reply:
updated!